draft-ietf-rats-corim.md

```diff
@@ -1123,26 +1123,38 @@ the object relates to the subject.
 #### Device Identity Triple {#sec-comid-triple-identity}
 
 A Device Identity triple relates one or more cryptographic keys to a device.
-The subject of an identity triple uses an instance or class identifier to refer
-to a device, and a cryptographic key is the object. The predicate asserts that
-at least one of the keys, as defined by `$crypto-key-type-choice`, authenticates the Attester.
+The triple subject is an identity used to name a device.
```

Comment:
Just a nit: maybe we could add a sentence to clarify the distinction between DI and AK triples?

Reply:
> Just a nit: maybe we could add a sentence to clarify the distinction between DI and AK triples?

see suggested text at line 1158